### PR TITLE
Updated to work with new gcloud SDK

### PR DESCRIPTION
**Pull request by elibixby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,24 +28,10 @@ Here is how to configure the Compute Engine firewall.
...
+##Deploy
 
-    mvn help:describe -Dplugin=appengine
+After setting up Maven, you can deploy your app locally, or to the Google Cloud Platform as described [here](https://cloud.google.com/appengine/docs/java/managed-vms/maven#run_and_deploy_your_app_with_the_cloud_sdk_development_server), then visit `http://chat.your-app-id.appspot.com/`.
 
-[1]: https://docs.google.com/document/d/1VH1oVarfKILAF_TfvETtPPE3TFzIuWqsa22PtkRkgJ4
-[2]: https://groups.google.com/forum/?fromgroups#!topic/google-appengine/gRZNqlQPKys
+[1]: https://cloud.google.com/appengine/docs/java/managed-vms/
+[2]: https://cloud.google.com/appengine/docs/java/managed-vms/#install-sdk
 [3]: https://cloud.google.com/console
```
Comment (tmatsuo) [inline README.md]: Is it possible to update the cloud console link to:
https://console.developers.google.com/
?

Although I'm OK for me to create another CL for this.

It's off-topic, but I think we'd better automate this kind of change.

File: pom.xml [hunk 2]
```diff
@@ -115,11 +115,11 @@
         <artifactId>maven-war-plugin</artifactId>
         <version>2.3</version>
         <configuration>
-          <archiveClasses>true</archiveClasses>
+  	  <archiveClasses>true</archiveClasses>
```
Comment (tmatsuo) [inline pom.xml]: What is this change for?

File: pom.xml [hunk 3]
```diff
@@ -132,6 +132,13 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>${appengine.target.version}</version>
       </plugin>
+
+      <plugin>
+      	<groupId>com.google.appengine</groupId>
+	<artifactId>gcloud-maven-plugin</artifactId>
+      	<version>1.9.17</version>
```
Comment (tmatsuo) [inline pom.xml]: How about to update the value in $appengine.target.version in properties section and use ${appengine.target.version} here?

**Comment by tmatsuo** [comment]
Thanks for the update!

File: pom.xml (hunk 2 above)
Comment (elibixby) [inline pom.xml]: Ahh nothing this was just troubleshooting when I was having maven problems. Will re-add

File: README.md (hunk 1 above)
Comment (elibixby) [inline README.md]: Definitely interested in trying to automate parts of github readme's in general.

File: pom.xml (hunk 2 above)
Comment (tmatsuo) [inline pom.xml]: Just a nit but I think the whitespace contains tab? Can you untabify if so?

**Comment by tmatsuo** [comment]
Just having a nit, but otherwise LGTM. I recommend you squash the changes and merge it as a single commit.

**Comment by elibixby** [comment]
TODO: need to add a step in `README.md` which explicitly describes how to make the gcloud-maven-plugin point to the correct directory for the gcloud SDK. Currently assuming users put the gcloud SDK in their home directory. This may not be the case.

**Comment by elibixby** [comment]
Waiting on updates to gcloud-maven-plugin, as well a couple internal fixes that were discovered in the process.

**Comment by tmatsuo** [comment]
How do you want to proceed?

**Comment by elibixby** [comment]
Looks like this PR is out of date. (I honestly have no memory of making it but it looks like ludo has already made the necessary changes). Closing.

**Pull request closed without merging**